src/runtime/ModuleStore.js

```diff
@@ -152,9 +152,12 @@
               evaled.push(frame);
             }
           });
-          ex.stack = evaled.join('\n');
+          // Allocate new object, some platforms make ex.stack read-only.
+          ex = {
```

Comment:
Looks like it is configurable to me:

```js
Object.getOwnPropertyDescriptor(new Error, 'stack')
// Object {get: function, set: function, enumerable: false, configurable: true}
```

(I also tried in Firefox).

So, it seems like we should at least do a `new Error` and then reconfigure the stack `property`.